### PR TITLE
New Presence (AnimeX)

### PR DESCRIPTION
**Pull request by veselysps**

### CODE REVIEW

**Comment by veselysps** [comment]
Better screenshots:
Browsing:

Video paused:

Video playing:

**Comment by veselysps** [comment]
New screenshots:
Paused

File: websites/A/AnimeX.tech/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "description": {
+    "en": "AnimeX is a free anime streaming site.",
+    "ro": "AnimeX este un site de streaming gratuit pentru anime.",
+    "pt_BR": "AnimeX é um site de streaming de animes gratuito.",
+    "fr": "AnimeX est un site de streaming d'animés gratuit.",
+    "uk_UA": "AnimeX - безкоштовний веб-сайт для стрімінгу аніме.",
+    "nl": "AnimeX is een gratis anime streaming-site."
+  },
+  "url": "animex.tech",
+  "version": "1.1.2",
```
Comment (Bas950) [inline websites/A/AnimeX.tech/dist/metadata.json:16]: version 1.0.0 on release

File: websites/A/AnimeX.tech/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "author": {
+    "name": "NotMarek",
+    "id": "206299359007080450"
+  },
+  "service": "AnimeX",
```
Comment (Bas950) [inline websites/A/AnimeX.tech/dist/metadata.json:6]: service name and folder name must be the same

File: websites/A/AnimeX.tech/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "description": {
+    "en": "AnimeX is a free anime streaming site.",
+    "ro": "AnimeX este un site de streaming gratuit pentru anime.",
+    "pt_BR": "AnimeX é um site de streaming de animes gratuito.",
+    "fr": "AnimeX est un site de streaming d'animés gratuit.",
+    "uk_UA": "AnimeX - безкоштовний веб-сайт для стрімінгу аніме.",
+    "nl": "AnimeX is een gratis anime streaming-site."
```
Comment (Bas950) [inline websites/A/AnimeX.tech/dist/metadata.json:13]: I bet you dont speak all these languages, only add descriptions for the languages you can speak.

File: websites/A/AnimeX.tech/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "url": "animex.tech",
+  "version": "1.1.2",
+  "logo": "https://i.kawaii.sh/63~NXHP.png",
+  "thumbnail": "https://i.kawaii.sh/HIkfJC8.png",
```
Comment (Bas950) [inline websites/A/AnimeX.tech/dist/metadata.json:18]: upload to imgur

File: websites/A/AnimeX/dist/metadata.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "NotMarek",
+    "id": "206299359007080450"
+  },
+  "service": "AnimeX",
+  "description": {
+    "en": "AnimeX is a free anime streaming site without ads.",
+    "cs": "AnimeX je bezplatná stránka pro streamování anime bez reklam."
```
Comment (Bas950) [inline websites/A/AnimeX/dist/metadata.json:9]: cs_CZ

Comment (veselysps) [reply websites/A/AnimeX/dist/metadata.json:9]: Deepscan denied that

Comment (Bas950) [reply websites/A/AnimeX/dist/metadata.json:9]: deepscan can't deny that.

Comment (veselysps) [reply websites/A/AnimeX/dist/metadata.json:9]: Oh sorry I had cs-CZ I will make a commit tommorow

Comment (Bas950) [reply websites/A/AnimeX/dist/metadata.json:9]: ah yea the - is blocked since its _ thats why it was denied.

**Pull request merged**